### PR TITLE
Full Website PR

### PR DESCRIPTION
**Pull request by Kossi-stack**
In this pull request the following requirements are: 

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Added style using inline styles and importing CSS
- Created a route for every page.
- Navigation links for all 3 routes are displayed on each page.
- Used React-Router to make the routes work

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  background: #e9d5d5;
+  padding: 2rem;
+  border-radius: 2rem;
+  line-height: 1.6rem;
+}
+
+.container h3 {
+  background: #e9d5d5;
+  padding: 1.5rem;
+  margin: 3rem 57rem 3rem 2rem;
```
Comment (henatan99) [inline src/App.css]: - [ ] I think the right margin is unreasonably big and it affects the component display. Kindly make changes here. :pray: 

image

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        It is important to be aware of the character of the sources for the study of the history of mathematics.
+        The history of Mesopotamian and Egyptian mathematics is based on the extant original documents written by scribes.
+        Although in the case of Egypt these documents are few, they are all of a type and leave little doubt that Egyptian mathematics was, on the whole,
+        elementary and profoundly practical in its orientation. For Mesopotamian mathematics, on the other hand, there are a large number of clay tablets,
+        which reveal mathematical achievements of a much higher order than those of the Egyptians. The tablets indicate that the Mesopotamians had a great deal of remarkable mathematical knowledge,
+        although they offer no evidence that this knowledge was organized into a deductive system. Future research may reveal more about the early development of mathematics in Mesopotamia or about its influence on Greek mathematics,
+        but it seems likely that this picture of Mesopotamian mathematics will stand.Ancient mathematical sources.
+      </p>
+    </div>
+  </Route>
```
Comment (henatan99) [inline src/pages/Home.js]: - [ ] I think the page routing is done in the `App` component and you don't need routing here. Kindly remove the routing here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Kossi-stack , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Kossi-stack, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**